subprojects/gradle/instrumentation-tests/src/main/kotlin/com/avito/instrumentation/reservation/client/kubernetes/KubernetesReservationClient.kt

```diff
@@ -320,6 +330,8 @@ class KubernetesReservationClient(
 
         object Idling : State()
     }
+
+    companion object
```

Comment:
What is this?

Reply:
my "signature" companion object for `createStubInstance()` function in test fixtures module